swift/ql/test/query-tests/Security/CWE-094/UnsafeJsEval.swift

```diff
@@ -175,17 +175,17 @@ func testAsync(_ sink: @escaping (String) async throws -> ()) {
 		let url = URL(string: "http://example.com/")
 
 		try! await sink(localString) // GOOD: the HTML data is local
-		try! await sink(try String(contentsOf: URL(string: "http://example.com/")!)) // BAD [NOT DETECTED - TODO]: HTML contains remote input, may access local secrets
-		try! await sink(try! String(contentsOf: url!)) // BAD [NOT DETECTED - TODO]
+		try! await sink(try String(contentsOf: URL(string: "http://example.com/")!)) // $ MISSING: Alert (HTML contains remote input, may access local secrets)
+		try! await sink(try! String(contentsOf: url!)) // $ MISSING: Alert
```

Comment:
Could we keep the `TODO`s both here and further below. It also looks like you've not been completely consistent. Here you've put the reason between parentheses, but in similar cases below you haven't.

Reply:
I kinda thought the `MISSING` annotation played the role of a TODO?

Reply:
I think the `TODO` meant someone was meaning to fix it imminently.  But this is out-of-date, these are not a priority right now, so I think it's fine that we lose that.

Reply:
SGTM.